taskcat/_amiupdater.py

```diff
@@ -494,8 +382,8 @@ def update_amis(self):
                     _write_template = True
             if _write_template:
                 template.write()
-        LOG.info("Templates updated as necessary")
         if _write_template:
+            LOG.info("Templates updated")
             raise AMIUpdaterCommitNeededException
 
-        LOG.info("Complete!")
+        LOG.info("No AMI's needed updates.")
```

Comment:
This will log every time, immediately after line 386.

Reply:
in my tests this only runs when `raise AMIUpdaterCommitNeededException` isn't called. Is it not correct that the absence of that raise indicates there was nothing updated ?